examples/gno.land/r/stefann/home/home.gno

```diff
@@ -0,0 +1,284 @@
+package home
+
+import (
+	"sort"
+	"std"
+	"strings"
+
+	"gno.land/p/demo/ownable"
+	"gno.land/p/demo/ufmt"
+
+	"gno.land/r/stefann/registry"
+)
+
+type City struct {
+	Name string
+	URL  string
+}
+
+type Sponsor struct {
+	Address std.Address
+	Amount  std.Coins
+}
+
+type Profile struct {
+	pfp     string
+	aboutMe []string
+}
+
+type Travel struct {
+	cities           []City
+	currentCityIndex int
+	jarLink          string
+}
+
+type Sponsorship struct {
+	maxSponsors    int
+	sponsors       map[std.Address]std.Coins
```

Comment:
You should utilize an `*avl.Tree`, instead of a `map` in Gno 🙏 
https://docs.gno.land/concepts/effective-gno/#using-avltree-for-efficient-data-retrieval

Reply:
Fixed! 6cb50b71b99b5c88937dc760437934302c89c9b0

I'm not sure if there are better practices for using an AVL tree, so feel free to point out any improvements or corrections if I've used it incorrectly